mangaki/mangaki/static/css/skin.css

```diff
@@ -374,13 +279,352 @@ ul.checklist .list-icon {
     font-size: 1.15em;
 }
 
-/** Anime **/
-/** Manga **/
+/**************
+ * WORK CARDS *
+ **************/
+
+.work-card {
+  display: flex;
+  flex-direction: column;
+
+  /* Cosmetics */
+  background: #fff;
+  border: none;
+  border-radius: 4px;
+  box-shadow: 0 1px 3px 0 #d4d4d5, 0 0 0 1px #d4d4d5;
+  overflow: hidden;
+
+  /* Create a new stacking context for the work-cover's blurred background. */
+  position: relative;
+  z-index: 0;
+
+  height: 370px;
+  width: 320px;
+
+  max-height: 90vh;
+  max-width: 90vw;
+}
+
+.work-card__media {
+  height: 100%;
+  width: 100%;
+}
+
+.work-cover {
+  overflow: hidden;
+
+  position: relative;
+
+  background: center center no-repeat;
+  background-size: 100% auto;
+}
+
+.work-cover_backdrop {
+  background-size: 0;
+}
+
+.work-cover:before, .work-cover_backdrop:after {
+  content: '';
+  display: block;
+
+  background: center center no-repeat;
+  background-image: inherit;
+
+  position: absolute;
+
+  height: 100%;
+  width: 100%;
+}
+
+.work-cover:before {
+  z-index: -1;
+  filter: opacity(50%) blur(10px);
+  background-size: cover;
+}
+
+.work-cover_backdrop:after {
+  background-size: 100% auto;
+  /* transition: filter .5s ease-in; */
+  will-change: filter;
+}
+
+.work-cover__wrapper {
+  position: absolute;
+  color: #fff;
+
+  width: 100%;
+  z-index: 1;
+
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.work-cover__title {
+  flex: 0 0 auto;
+  width: 100%;
+  padding: 8px 12px;
+
+  word-wrap: break-word;
+  hyphens: auto;
+
+  background: rgba(0, 0, 0, 0.6);
+
+  margin-top: auto;
+}
+
+.work-cover_backdrop:hover:after {
+  filter: opacity(50%) blur(5px);
+  /* transition: filter .5s ease-out; */
```

Comment:
We have to determine whether we want the animation or not (or possibly only on small screens). I kept it commented for now so that we can easily try out both options.